app/assets/javascripts/discourse/app/controllers/topic.js

```diff
@@ -1700,25 +1700,37 @@ export default Controller.extend(bufferedProperty("model"), {
             postNumberDifference > 0 &&
             postNumberDifference < 7
           ) {
-            this._scrollToPost(data.post_number);
+            this._scrollToDiscobotPost(data.post_number);
           }
         }
       },
       this.get("model.message_bus_last_id")
     );
   },
 
-  _scrollToPost(postNumber) {
+  _scrollToDiscobotPost(postNumber) {
```

Comment:
Looks like this code shouldn't be in core anymore. When I added this code back then, it was supposed to apply to all PMs but it looks like that changed in https://github.com/discourse/discourse/commit/267d129f38d6d45d0863014bf93ad1bf932c70b3.